### PR TITLE
[APPAI-1357] Avoid transitive dependency scan for stack analyses v2

### PR DESCRIPTION
**Pull request by dgpatelgit**
# Description

Avoid transitive dependency scan for stack analyses v2

Fixes # (issue)

https://issues.redhat.com/browse/APPAI-1357

## Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: integration-tests/features/steps/stack_analyses_v2.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def post_request(context, ecosystem, manifest, with_user_key, with_valid_user_ke
 
     # set default values
     files = {}
-    data = {}
+    data = {'show_transitive': 'false'}
```
Comment (yzainee) [inline integration-tests/features/steps/stack_analyses_v2.py:54]: @dgpatelgit can you please point me to the code in api server which enables / disabled the transitive data on the basis of this flag

Comment (dgpatelgit) [reply integration-tests/features/steps/stack_analyses_v2.py:54]: > @dgpatelgit can you please point me to the code for v2 in api server which enables / disables the transitive data on the basis of this flag

Its under stack_analyses.py -> _make_backbone_request() below is req. params send to backbone

**Comment by dgpatelgit** [comment]
> BTW, Though it is disabled by default in V1, can you please explicitly disable the same there as well?

This is done, PTAL

**Comment by sawood14012** [comment]
@dgpatelgit can you please squash your commits and push

**Pull request merged**